webium/base_page.py

```diff
@@ -21,9 +21,15 @@ def _get_driver():
     _get_driver().implicitly_wait(timeout)
     try:
         def is_displayed():
-            element = getattr(self, element_name, None)
-            if not element:
+            try:
+                element = getattr(self, element_name)
+            except AttributeError:
                 raise WebiumException('No element "{0}" within container {1}'.format(element_name, self))
+            if isinstance(element, list):
+                if element:
```

Comment:
Well, ```all([])``` produces ```True```. So, it depends on what you expect from Finds which does not contain any visible elements